_info/Tribute

```diff
@@ -1,6 +1,7 @@
 Tribute
 
-Every morning, the king has to choose to tribute one of this team members. This is mandatory. The king has to also choose a certain amount of money to assign to the person. Each team will do the same. The team which pays the least money for their tribute, will have their tribute killed. Use `!tribute` to set your tribute candidate if you're the king. 
+Every morning, the king has to choose to tribute one of this team members. This is mandatory. The king has to also choose a certain amount of money to assign to the person. Each team will do the same. 
+The team which pays the least money for their tribute, will have their tribute killed. Use `!tribute` to set your tribute candidate if you're the king. 
 If a team forgets to set a tribute, a random person from the team will be auto tributed for 0 coins. This can sometimes auto kill the king, so always set a tribute. 
```

Comment:
again, try to explain things in the fewest words possible whenever possible. very wordy explanations are unideal

Reply:
I don't know which part of the sentence was lengthy, but I'll shorten it